Directory.Packages.props

```diff
@@ -10,13 +10,12 @@
     <PackageVersion Include="FluentAssertions" Version="6.12.0" />
     <PackageVersion Include="FluentValidation.DependencyInjectionExtensions" Version="11.9.0" />
     <PackageVersion Include="MediatR" Version="12.2.0" />
-    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="8.0.0" />
-    <PackageVersion Include="Microsoft.AspNetCore.OpenApi" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="8.0.1" />
+    <PackageVersion Include="Microsoft.AspNetCore.OpenApi" Version="8.0.1" />
     <PackageVersion Include="Microsoft.EntityFrameworkCore" Version="7.0.5" />
     <PackageVersion Include="Microsoft.Extensions.DependencyInjection" Version="8.0.0" />

```

Comment:
**Suggestion:** Consider keeping the package versions consistent across the project. The version of `Microsoft.AspNetCore.Mvc.Testing` and `Microsoft.AspNetCore.OpenApi` has been updated to `8.0.1` while `Microsoft.Extensions.DependencyInjection` is still at `8.0.0`. If there is no specific reason for this, it would be better to keep them at the same version. [best practice]
```suggestion
    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="8.0.1" />
    <PackageVersion Include="Microsoft.AspNetCore.OpenApi" Version="8.0.1" />
    <PackageVersion Include="Microsoft.Extensions.DependencyInjection" Version="8.0.1" />
```